Yoast/wordpress-seo
# Remove Google+ and replace it with other stuff

## Summary

This PR can be summarized in the following changelog entry:

* Removes Google+ from the various interfaces: social accounts, user profiles and more.
* Adds a Wikipedia URL field to the social accounts list, to be used in sameAs Schema.org output.
* Adds a LinkedIn profile URL field to user profiles.

## Relevant technical choices:

* Updated the above in admin pages and in the config wizard.

## Test instructions

This PR can be tested by following these steps:

* See the Google+ fields are gone on admin and user profile.
* See the new LinkedIn field on user profile and that it works.
* See the new Wikipedia field on the Social -> Accounts list and that it works.

## UI changes

## Quality assurance


Fixes #11191
